src/components/Button.js

```diff
@@ -71,7 +73,8 @@ export const Button = ({
         !invert && isDelete && styles.rejectButton,
         big && [styles.bigButton, styles.bigButtonRadius],
         small && [styles.smallButton, styles.smallButtonRadius],
-        smallest && [styles.smallestButton, styles.smallestButtonRadius]
+        smallest && [styles.smallestButton, styles.smallestButtonRadius],
+        invert && lightest && [styles.buttonInvert, styles.lightestInvert]
```

Comment:
why checking `invert` again here? `invert` and `lightest` should be seen independent of each other. if we use both props, both will be applied. `styles.buttonInvert` is already there and `lightest` should work like all the others with explicit `lightestBorderColor` here.

Reply:
I made a new force-commit and applied the changes you mentioned with this commit.